recipe/build.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 set -ex
 
-if [[ $(uname) == Darwin ]]; then
+# if [[ $(uname) == Darwin ]]; then
     # -dead_strip_dylibs causes the tests in make check to fail to find
     # the default_material global variable in libctlgeom (from libctl)
-    export LDFLAGS="${LDFLAGS/-Wl,-dead_strip_dylibs/}"
-fi
+#     export LDFLAGS="${LDFLAGS/-Wl,-dead_strip_dylibs/}"
+# fi
```

Comment:
do you remember how to check if this is actually necessary? i feel like the fewer flags the better here.

Reply:
great thanks.